### ISSUE TITLE
[1.10.2] [Major Crash] Using 12.18.2.2125 the game fails to start

### ISSUE THREAD

**Issue by SnowShock35** [squeek502/AppleCore#81]
Hi squeek,

I've encountered this crash with apple core using the latest version of Forge (12.18.2.2125) Could you take a look at it? 

Thanks. :) 

P.S. Sorry for the, lack of a better word, shit report. I've not been feeling to good. Apologies.

```
2016-11-12 22:53:43,918 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
2016-11-12 22:53:43,919 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
[22:53:43] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
[22:53:43] [main/INFO] [LaunchWrapper]: Using primary tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
[22:53:43] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLTweaker
[22:53:43] [main/INFO] [FML]: Forge Mod Loader version 12.18.2.2125 for Minecraft 1.10.2 loading
[22:53:43] [main/INFO] [FML]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_25, running on Windows 10:amd64:10.0, installed at D:\Minecraft\Launchers\Curse\Minecraft\Install\runtime\jre-x64\1.8.0_25
2016-11-12 22:53:44,016 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
2016-11-12 22:53:44,026 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
2016-11-12 22:53:44,027 WARN Unable to instantiate org.fusesource.jansi.WindowsAnsiOutputStream
[22:53:44] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLInjectionAndSortingTweaker
[22:53:44] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLDeobfTweaker
[22:53:44] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLInjectionAndSortingTweaker
[22:53:44] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLInjectionAndSortingTweaker
[22:53:44] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModManager$FMLPluginWrapper
[22:53:45] [main/INFO] [FML]: Found valid fingerprint for Minecraft Forge. Certificate fingerprint e3c3d50c7c986df74c645c0ac54639741c90a557
[22:53:45] [main/INFO] [FML]: Found valid fingerprint for Minecraft. Certificate fingerprint cd99959656f753dc28d863b46769f7f8fbaefcfc
[22:53:45] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModManager$FMLPluginWrapper
[22:53:45] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.FMLDeobfTweaker
[22:53:45] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.relauncher.CoreModManager$FMLPluginWrapper
[22:53:45] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher.TerminalTweaker
[22:53:45] [main/INFO] [LaunchWrapper]: Calling tweak class net.minecraftforge.fml.common.launcher.TerminalTweaker
[22:53:45] [main/INFO] [LaunchWrapper]: Launching wrapped minecraft {net.minecraft.client.main.Main}
[22:53:45] [main/INFO] [STDOUT]: [net.minecraft.client.main.Main:main:56]: Completely ignored arguments: [--nativeLauncherVersion, 307]
[22:53:45] [Client thread/INFO]: Setting user: SnowShock35
[22:53:46] [Client thread/ERROR] [LaunchWrapper]: Unable to launch
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_25]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_25]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.lang.VerifyError: Expecting a stackmap frame at branch target 49
Exception Details:
  Location:
    net/minecraft/block/BlockCrops.func_180650_b(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/block/state/IBlockState;Ljava/util/Random;)V @25: if_acmpeq
  Reason:
    Expected stackmap frame at this location.
  Bytecode:
    0x0000000: 2a2b 2c2d 1904 b700 7d2a 2b2c 2d19 04b8
    0x0000010: 0083 3a07 1907 b200 89a5 0018 1907 b200
    0x0000020: 8ca6 0077 2b2c b600 92b6 0098 1009 a100
    0x0000030: 6a2a 2db6 009a 3605 1505 2ab6 0079 a200
    0x0000040: 5a2a 2b2c b800 9e38 0619 07b2 0089 a500
    0x0000050: 1e19 07b2 008c a600 212b 2c2d 1904 129f
    0x0000060: 1706 6e8b 0460 b600 a59a 000e 04a7 000b
    0x0000070: 2a2b 2c2d b800 a903 b800 af99 001d 2b2c
    0x0000080: 2a15 0504 60b6 00b1 05b6 00b5 572b 2c2d
    0x0000090: 2b2c b600 b9b8 00bd b1                 
  Stackmap Table:
    full_frame(@119,{Object[#2],Object[#148],Object[#142],Object[#39],Object[#161],Integer,Float},{Object[#148],Object[#142],Object[#39]})
    full_frame(@120,{Object[#2],Object[#148],Object[#142],Object[#39],Object[#161],Integer,Float},{Object[#148],Object[#142],Object[#39],Integer})
    chop_frame(@152,2)

	at net.minecraft.block.Block.func_149671_p(Block.java:2165) ~[akf.class:?]
	at net.minecraft.init.Bootstrap.func_151354_b(Bootstrap.java:537) ~[kq.class:?]
	at net.minecraft.client.Minecraft.<init>(Minecraft.java:341) ~[bcx.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) ~[Main.class:?]
	... 6 more
Exception in thread "Client thread" [22:53:46] [Client thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: net.minecraftforge.fml.relauncher.FMLSecurityManager$ExitTrappedException
[22:53:46] [Client thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at net.minecraftforge.fml.relauncher.FMLSecurityManager.checkPermission(FMLSecurityManager.java:49)
[22:53:46] [Client thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.lang.SecurityManager.checkExit(SecurityManager.java:761)
[22:53:46] [Client thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.lang.Runtime.exit(Runtime.java:107)
[22:53:46] [Client thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at java.lang.System.exit(System.java:968)
[22:53:46] [Client thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:138)
[22:53:46] [Client thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by GirafiStudios** [comment]
Duplicate of https://github.com/squeek502/AppleCore/issues/80
Squeek have already fixed it, so I assume he will release a version with it soon :)

EDIT: Use Forge version 12.18.2.2124 for now until a new version of Apple Core is out!

**Comment by SnowShock35** [comment]
Sorry Girafi :heart:

**Comment by squeek502** [comment]
Should be fixed in v2.1.0: https://github.com/squeek502/AppleCore/releases/tag/v2.1.0
